plugins/task/testtasks/testtasks.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * A test plugin for com_scheduler.
```

Comment:
Probably you could rename this plugin to "demotask" and really comment everything, then we could ship it as help for other developer.

Reply:
Done!